app_basics/commands/util.py

```diff
@@ -0,0 +1,42 @@
+def file_tail_or_head(f_name: str, mode: int, n_lines: int = 3):
+    """
+    Function to read first or last n_lines lines from a file
+    f_name - the file relative path
+    mode - 1 is head, -1 is tail
+    n_lines - Number of lines to read
+    """
+
+    with open(f_name, "r") as file:
+        if mode == 1:
+            for line in (file.readlines()[:n_lines]):
+                print(line, end='')
+        elif mode == -1:
+            for line in (file.readlines()[-n_lines:]):
+                print(line, end='')
+        else:
+            print("Supports only mode 1/-1")
+            return -1
+
+    print("")
+
+
+def csv_tail_or_head(csv: str, mode: int, n_columns: int = 3):
```

Comment:
try to simplify this function and make it more reusable

Reply:
Can't see how can simplify this function, by its definition is very specific to do tail or head on a line in CSV format.
Still add minor changes, like 'elif' replaced with 'else', and add default value 1 to mode.